modules/vshn-lbaas-cloudscale/variables.tf

```diff
@@ -83,3 +83,9 @@ variable "internal_vip" {
   description = "Virtual IP for the Kubernetes/OpenShift API in the internal network"
   default     = ""
 }
+
+variable "team" {
+  type        = string
+  description = "Team to assign the load balancers to in Icinga. All lower case."
+  default     = ""
```

Comment:
I'd prefer making this mandatory as well, we can always consider making it optional in the Terraform which uses this variable.

```suggestion
```

Reply:
Good point!